src/uct/ib/base/ib_iface.h

```diff
@@ -175,7 +175,12 @@ struct uct_ib_iface_config {
 
 
 enum {
+#if UCS_ENABLE_ASSERT
+    /* Check CQ overrun if assert mode enabled. */
+    UCT_IB_CQ_IGNORE_OVERRUN         = 0,
```

Comment:
we can define the flag anyway and just not pass it in assert mode

Reply:
yes, but his will be more changes

Reply:
forgot to explore this option. Will check